### PR TITLE
config is not in /etc/georchestra/datadir

### PR DESCRIPTION
**Pull request by MaxiReglisse**
config files are not in /etc/georchestra/datadir but in  /etc/georchestra

### CODE REVIEW

**Comment by fvanderbiest** [comment]
If  `/etc/georchestra` does not exist yet, `sudo mv datadir /etc/georchestra` creates the directory as expected

**Comment by MaxiReglisse** [comment]
you are right...
but so, the same command could give different results, according that the directory exists or not. my opinion is that it is not good thing. i would prefer : 
mkdir /etc/georchestra
cp datadir/* /etc/georchestra/.

File: docs/docker.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Feel free to comment out the apps you do not need.
 Create a configuration directory on your host machine:
 ```
 git clone -b docker-master https://github.com/georchestra/datadir.git
-sudo mv datadir /etc/georchestra
+sudo mv datadir/* /etc/georchestra/.
```
Comment (Vampouille) [inline docs/docker.md]: Can you add mkdir command ?

Comment (MaxiReglisse) [inline docs/docker.md]: it seems i cannot modify the page... perhaps because the issue is closed ?

**Pull request closed without merging**